### DISCUSSION TITLE
2023/03/13/%E7%AC%AC%E4%BA%94%E5%B1%8A%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E9%9D%92%E8%AE%AD%E8%90%A5%E9%A1%B9%E7%9B%AE%E6%80%BB%E7%BB%93/

### DISCUSSION THREAD

**Discussion by giscus** [nickxudotme/nickxudotme.github.io#3] [General]
# 2023/03/13/%E7%AC%AC%E4%BA%94%E5%B1%8A%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E9%9D%92%E8%AE%AD%E8%90%A5%E9%A1%B9%E7%9B%AE%E6%80%BB%E7%BB%93/

PS：以后写感想类博客一定要当天冲完，不然拖个一个星期真的找不到那种劲了  首先我真的想向字节跳动和掘金社区道个歉，这次青训营我主要就是奔着大项目来的 ，平时的课基本没怎么听 关于课的话，其实是挺好的，但是和第三届的重复了很多，加之一开始结营条件定的太高了，考虑到寒假的精力，我选择直接放弃结营证书，全力去冲项目证书了 （结果最后告诉我没有实体的项目证书） 好了，回到正题，关于这次的项目总结，关于

https://nickxu.me/2023/03/13/%E7%AC%AC%E4%BA%94%E5%B1%8A%E5%AD%97%E8%8A%82%E8%B7%B3%E5%8A%A8%E9%9D%92%E8%AE%AD%E8%90%A5%E9%A1%B9%E7%9B%AE%E6%80%BB%E7%BB%93/

**Comment by meiyu0714** [comment]
写的真的非常好。就是背景太黑了。眼睛看了一会儿很难受 希望有调白天模式。

**Reply by nickxudotme** [reply]
可以的，没问题哦 :+1:
